#!/usr/bin/env python3
PACKAGE = "spring_line_sim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# param, data_type, 0, description, defaut, min, max
gen.add("hue_l", double_t, 0, "hue low", 25, 0, 255)
gen.add("hue_h", double_t, 0, "hue high", 35, 0, 255)

gen.add("sat_l", double_t, 0, "sat low", 0, 0, 255)
gen.add("sat_h", double_t, 0, "sat high", 255, 0, 255)

gen.add("val_l", double_t, 0, "val low", 0, 0, 255)
gen.add("val_h", double_t, 0, "val high", 255, 0, 255)

# the third parameter must match this file's name
exit(gen.generate(PACKAGE, "detect_yellow", "DetectYellow"))
